hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAdmin1.java

```diff
@@ -1492,4 +1487,16 @@ public void testMergeRegions() throws Exception {
       this.admin.deleteTable(tableName);
     }
   }
+
+  @Test (timeout=30000)
+  public void testTableNotFoundException() throws Exception {
+    ZooKeeperWatcher zkw = HBaseTestingUtility.getZooKeeperWatcher(TEST_UTIL);
```

Comment:
connection leak, needs to be closed.

Instead use the non-static version TEST_UTIL.getZookeeperWatcher()... that is automatically closed with HTU teardown.

Reply:
Thank you for the feedback. Addressed in latest commit. Please review again.